### ISSUE TITLE
{Freeway-Bug} [Weather Card]: 'Weather card' is not properly visible after resizing the application under 'Weather' pane.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#7875]
### Target Platforms

iOS

### SDK Version

Version 1.0(2.4.0-beta.22.9.4.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 15.6.1
Application: Adaptive Cards
App Version: Version 1.0(2.4.0-beta.22.9.4.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Scenarios' control and activate it.
6. Navigate to 'Weatherlarge.Json' under 'Scenario' list item and activate it.
7. Card will be appear.
8. Observe the issue after resizing the screen.

**Actual Result:**
After applying larger text setting, Temperature under weather card is getting cropped and not visible properly.

**Expected Result:**
After applying larger text setting temperature under weather card should be visible properly.
 
**Notes:**
1. Same issue is repro under 'Flight Details' screen. Attachment is added for the reference. (Note- Destination text is getting cropped).
2. Same issue is repro under 'Food Order' screen for the 'I am allergic to...' text. Attachment is added (Note - Text under Food Order). 

**User Experience:**
If after enlarging the text size of application, some content or controls are not properly appearing, then it would impact the low vision users.

**Mas Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.04.04

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

![Larger mode](https://user-images.githubusercontent.com/95913748/190461347-b221d413-3da0-4a7e-b8c0-dda6d03d435b.PNG)
![Normal Mode](https://user-images.githubusercontent.com/95913748/190461361-66012358-f553-4a04-bef9-85701d23ed50.PNG)
![Note- Destination text is getting cropped](https://user-images.githubusercontent.com/95913748/190462951-c612322d-664b-41ed-8100-42c784cee109.PNG)
![Note - Text under Food Order](https://user-images.githubusercontent.com/95913748/190464622-d67a3651-4463-43d2-85a8-a2dfdc979e54.PNG)

### Card JSON

```JSON
NA
```
